src/__init__.py

```diff
@@ -0,0 +1,22 @@
+
+from flask import Flask, url_for
+from flask_sqlalchemy import SQLAlchemy
+
+
+
+
+app = Flask(__name__)
+app.config['SECRET_KEY'] = "ADFAJKhnasdjfi890"
```

Comment:
Usually don't want to have these types of keys in the code, and could be set up as environment variables or have another service like Vault used to access these for better security. For now, it is fine, as we won't be deploying this. But in the future, let's avoid making this public.